### DISCUSSION TITLE
Matrix View not displaying nodes

### DISCUSSION THREAD

**Discussion by scottrlarson** [SkepticMystic/breadcrumbs#459] [Q&A]
I'm new to obsidian and the breadcrumbs plug-in and I wonder if I am missing something. I added (from what I understand) correct relationships to some of my documents to start creating a hierarchy, but Im not getting a display in any of the breadcrumb views, except the duck view which lists all my documents without any hierarch. I am I doing something wrong?
![Screenshot from 2022-11-06 09-56-40](https://user-images.githubusercontent.com/28913453/200187128-58a66f6a-f85c-402e-884e-2b92f1b94bc7.png)

**Comment by michaelpporter** [comment]
This was reported against the v3 Matrix View, which was fully rewritten in v4. The new Matrix renders incoming/outgoing edges grouped by field and no longer has this behaviour.

Closing as outdated — if you still see missing nodes on the current version, please open a new issue with your Breadcrumbs version and a small repro. 🍞
